app/src/main/java/fr/free/nrw/commons/depictions/SubClass/models/SubclassLabel.java

```diff
@@ -0,0 +1,64 @@
+package fr.free.nrw.commons.depictions.SubClass.models;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Model class for parsing SparqlQueryResponse
+ */
+public class SubclassLabel {
+
+    @SerializedName("type")
```

Comment:
javadoc: explanation and example value for each field

Reply:
added